packages/compile-solidity/compilerSupplier/loadingStrategies/Native.js

```diff
@@ -6,18 +6,13 @@ class Native extends LoadingStrategy {
   load() {
     const versionString = this.validateAndGetSolcVersion();
     const command = "solc --standard-json";
+    const maxBuffer = 1024 * 1024 * 10;
 
-    try {
-      return {
-        compile: options => String(execSync(command, { input: options })),
-        version: () => versionString
-      };
-    } catch (error) {
-      if (error.message === "No matching version found") {
-        throw this.errors("noVersion", versionString);
```

Comment:
Why'd you remove this invocation? Don't we need to handle this case? (@eggplantzzz do you know?)

Reply:
Unless I am missing something, this is returning an object and not executing anything, so there is no way that it will raise an exception at this point.
`compile` might fail later on but having the `try`/`catch` here will not help.
Simplified example:

```javascript
var a;
// try/catch not helping
try {
  a = () => { thrown RuntimeError('asdf') }
} catch (e) { console.log(e) }
a() // will not be caught
```

TBH I changed this mainly to please eslint so if you think it's better to keep this for another PR I can of course revert this change.

Reply:
Hm, I prefer leaving code alone unless the change is warranted. @eggplantzzz added that line and I suspect it does something important.

It'll definitely make this an easy merge at this point if you restrict this PR to just the max buffer change. Happy to figure out the error handling business in an issue or a subsequent PR (you could very well be right, but I just don't want to keep you waiting on this particular issue... would love to get your fix here out in this week's release!)

Reply:
Sure, sounds good.
Sorry for the back and forth, I think this is now as minimal as it gets in terms of changes.